src/basic/math/FIN.s

```diff
@@ -0,0 +1,151 @@
+;; #LAYOUT# STD *       #TAKE
+;; #LAYOUT# X16 BASIC_0 #TAKE-OFFSET 2000
+;; #LAYOUT# *   BASIC_0 #TAKE
+;; #LAYOUT# *   *       #IGNORE
+
+;
+; Math package - imports string to FAC1, ignores spaces
+;
+;
+; Input:
+; - .A needs to be loaded with first char of the string
+; - Carry flag must be clear
+; - string address in $7A/$7B
+;
+; See also:
+; - [CM64] Computes Mapping the Commodore 64 - page 116
+; - https://www.c64-wiki.com/wiki/Floating_point_arithmetic
+; - https://codebase64.org/doku.php?id=base:kernal_floating_point_mathematics
+;
+
+
+FIN_NUMDEC      = TEMPF2
+FIN_DIGIT       = TEMPF2 + 1
+FIN_DECFOUND    = TEMPF2 + 2
+FIN_EXPSIGN     = TEMPF2 + 3
+FIN_EXP         = TEMPF2 + 4
+
+
+FIN:
+    ldx #$00
```

Comment:
If it is only usable with 38K memory model, use `!ifdef CONFIG_MEMORY_MODEL_38K {` for conditional compilation and put a stub macro for other configs.